lib/manifestTools/assets/web-manifest.json

```diff
@@ -46,8 +46,8 @@
       "properties": {
         "density": {
           "description": "The density member of an icon is the device pixel density for which this icon was designed.",
-          "type": "number",
-          "default": 1.0
+          "type": [ "number", "string" ],
+          "default": [ 1.0, "one" ]

```

Comment:
Doh, no coffee for me this morning and this is what happens. Yes, should read "1.0". Resubmitting PR.